remme/tp/pub_key.py

```diff
@@ -61,18 +70,46 @@ def get_state_processor(self):
             }
         }
 
+    def _verify_rsa(self, ehs_bytes, eh_bytes, public_key, sig_padding):
```

Comment:
Well, I guess I have to rethink how we verify that the tx sender actually owns the key he's saving. When it comes to supporting different signatures the current way is painful.